go/common/viewingkey/viewing_key.go

```diff
@@ -66,6 +66,25 @@ type RPCSignedViewingKey struct {
 	SignatureWithAccountKey []byte
 }
 
+const (
+	pubKeyLen = 33
+	sigLen    = 65
+)
+
+func (vk RPCSignedViewingKey) Validate() error {
+	// todo - remove this when merging to main
+	if vk.Account == nil {
```

Comment:
We could make it compatible with the future format at this point I suppose, infer the account? Would be a throwaway code snippet but means main tools (auth client and gateway) can communicate with sepolia testnet.

Reply:
let's discuss in  planning. I don't think it's worth the effort